nrf52-hal-common/src/rng.rs

```diff
@@ -18,6 +18,7 @@ pub trait RngExt : Deref<Target=rng::RegisterBlock> + Sized {
 
 impl RngExt for RNG {
     fn constrain(self) -> Rng {
+        self.deref().config.write(|w| w.dercen().enabled());
```

Comment:
`.deref()` should not need to be called manually

Reply:
🤦‍♂️ fixed